EliteAPI.Tests/BindingTests.cs

```diff
@@ -0,0 +1,329 @@
+using System.Linq;
+using EliteAPI.Bindings;
+using FluentAssertions;
+using TUnit;
+
+namespace EliteAPI.Tests.Bindings;
+
+public class BindingParserTests
```

Comment:
could we add a test where we just read in a keybindings file and see it doesn't crash or anything?